src/main/java/placement/model/Board.java

```diff
@@ -0,0 +1,193 @@
+package placement.model;
+
+import placement.model.field.*;
+import placement.model.ship.Ship;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Represents battleship board in the context of ship placement (initial game phase).
+ */
+public final class Board {
```

Comment:
what is the test coverage for this class?

Reply:
100% :)